compiler/luci/import/src/GraphBuilder.cpp

```diff
@@ -48,19 +50,18 @@ void GraphBuilder::build(const circle::OperatorT &op, GraphBuilderContext *conte
   {
     const circle::TensorT &output_tensor = *tensors[outputs[0]];
     copy_tensor_attributes(output_tensor, node);
+    // mark shape_status
+    if (tensors_ptr->Get(outputs[0]) == nullptr)
```

Comment:
May I have a question? Is it impossible to get info like `tensors_ptr->Get(outputs[0]` from above `tensors`? like `tensors.at(outputs[0])`.

Reply:
Um.. existing code is incorrect. We need to check shape existance.
That is shoud be `if (tensors_ptr->Get(outputs[0])->shape() == nullptr)`

Reply:
Thank you for giving me a second chance to look at the code.